src/lib/aggregations/get-range.spec.ts

```diff
@@ -30,72 +31,135 @@ describe('getRangeAggregation', () => {
         }
     ]
 
-    it('accepts only schema field', () => {
-        const query = getRangeAggregation<HomeDocument>('address', ranges)
+    it('accepts only schema numeric field', () => {
+        const query = getRangeAggregation<HomeDocument>('propertyAreaSquared', ranges)
 
         expect(query).toEqual({
             range: {
-                field: 'address',
+                field: 'propertyAreaSquared',
                 ranges: [{ from: 10 }, { from: 15, to: 20 }, { to: 25 }]
             }
         })
     })
 
-    it('queries es for range aggregation with nested aggregation', async () => {
+    it('should queries elasticsearch for range aggregation ', async () => {
+        const service = app.get(ElasticsearchService)
+
+        const result = await service.search(HomeDocument, {
+            size: 0,
+            aggregations: {
+                testAggregation: getRangeAggregation('propertyAreaSquared', ranges)
+            }
+        })
+
+        const expectedResponseShape = [
+            {
+                key: '*-25.0',
+                to: 25,
+                doc_count: expect.any(Number)
+            },
+            {
+                key: '10.0-*',
+                from: 10,
+                doc_count: expect.any(Number)
+            },
+            {
+                key: '15.0-20.0',
+                from: 15,
+                to: 20,
+                doc_count: expect.any(Number)
+            }
+        ]
+
+        expect(result.aggregations.testAggregation.buckets).toEqual(expectedResponseShape)
+    })
+
+    it('should queries elasticsearch for range aggregation with nested aggregation', async () => {
         const service = app.get(ElasticsearchService)
         const result = await service.search(HomeDocument, {
             size: 10,
             aggregations: {
-                test: {
+                testAggregation: {
```

Comment:
we can change the name of `testAggregation` to `result`?

Reply:
then it would probably have to be changed in every test as well?

Reply:
just for this one, and for the next ones 🙏 - it works for the other ones, we can refactor it later as it is only aggregation name change ;)